core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2294,6 +2294,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_NETWORK_BLOCK_EVICTION_THREADS =
+      new Builder(Name.WORKER_NETWORK_BLOCK_EVICTION_THREADS)
+          .setDefaultSupplier(() -> Runtime.getRuntime().availableProcessors(),
```

Comment:
Is this a reasonable default?

Reply:
I found this to yield min eviction duration.

Reply:
Right, it pretty much allocates all resources on the machine to eviction if needed, that seems a bit aggressive?

Reply:
Should we leave one core out? Or half the core count is better?

Reply:
Half of the cores seems a more moderate choice, you can add some more details in the description about how this parameter can be tuned.